libs/wingsdk/src/ex/react-app.md

```diff
@@ -0,0 +1,87 @@
+---
+title: React Website
```

Comment:
Name consistency - shouldn't this match the .md file name? I'd expect both to be React App.
```suggestion
title: React App
```